### ISSUE TITLE
send-0019

### ISSUE THREAD

**Issue by xehpuk** [mailjet/mailjet-apiv3-dotnet#138]
See #69.

**Comment by nprather** [comment]
This might be unrelated, but I was getting this same error and instead of using a Marketing Template, I changed to using a Transaction Template and I think maybe that solved the problem... I hope.

**Comment by nelsonvarela** [comment]
Yeah, bug still very alive. Also have Transactional templates, but every time i save a template, the sender is back with its id,
